File: api/Practica.Common/Models/ClienteViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Practica.Common.Models
{
    public class ClienteViewModel
    {
        public int Clie_Id { get; set; }
        public string? Clie_DNI { get; set; }
        public string Clie_Nombre { get; set; }
        public string Clie_Apellido { get; set; }
        public string Clie_Sexo { get; set; }
        public int? Clie_Telefono { get; set; }
        public int? Esta_Id { get; set; }
        public string Muni_Id { get; set; }
        public int? Clie_Usua_Creacion { get; set; }
        public DateTime? Clie_Fecha_Creacion { get; set; }
        public int? Clie_Usua_Modifica { get; set; }
        public DateTime? Clie_Fecha_Modifica { get; set; }

        [NotMapped]
        public string Usua_Creacion { get; set; }

        [NotMapped]
        public string Usua_Modifica { get; set; }
    }
}

File: app/plantilla/src/app/demo/models/ClienteViewModel.ts
export class Cliente{
    clie_Id!: number;
    clie_DNI!:string;   
    clie_Nombre!: string;
    clie_Apellido!: string;
    clie_Sexo!:string;   
    esta_Descripcion!: string;
    clie_Telefono!: number;
    esta_Id!:number;   
    muni_Id!: string;
    dept_Id!: string;
}

export class Clienteddl{
    clie_Id!: number; 
    clie_Nombre!: string;
    clie_Apellido!: string;
}

export class Fill{
    clie_Id!: string;
    clie_DNI!:string;   
    clie_Nombre!: string;
    clie_Apellido!: string;
    clie_Sexo!:string;   
    esta_Descripcion!: string;
    usua_Creacion!:string;   
    clie_Fecha_Creacion!:string;   
    usua_Modifica!:string;   
    clie_Fecha_Modifica!:string; 
}